src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -109,34 +46,50 @@ export class FightOrFlight extends BuffWindow {
 		this.ignoreActions(EXCLUDED_ACTIONS.map(g => this.data.actions[g].id))
 
 		this.addEvaluator(new ExpectedGcdCountEvaluator({
-			expectedGcds: 11,
+			expectedGcds: 8,
 			globalCooldown: this.globalCooldown,
 			suggestionIcon: this.data.actions.FIGHT_OR_FLIGHT.icon,
 			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcds.content">
-				Try to land 11 physical GCDs during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
+				Try to land 8 GCDs during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
 			</Trans>,
 			suggestionWindowName,
 			severityTiers: SEVERITIES.MISSED_GCD,
 		}))
 
+		this.addEvaluator(new ExpectedActionsEvaluator({
+			expectedActions: [
+				{action: this.data.actions.GORING_BLADE, expectedPerWindow: 1},
+				{action: this.data.actions.CONFITEOR, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_FAITH, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_VALOR, expectedPerWindow: 1},
+				{action: this.data.actions.HOLY_SPIRIT, expectedPerWindow: 1},
```

Comment:
afaik this isn't a safe assumption - there will be times in the rotation where you'll end up with i.e. 3 atonements as filler unless you're doing spreadsheet optimisations. afaik current "general" recommended rotation burst window just locks in goring and confetti combo, and leaves the remaining 3 as priority based filler

Reply:
I don't agree with this because you should move around the Holy Spirit to fit in the window. It doesn't break combos and you can hold onto it and start a new combo if needed. Buffed Holy Spirits can and should always be worked into a Fight or Flight window

Reply:
I could have been a bit nicer with my previous comment. The Balance thought that was the case early on but it was proven to be a bad assumption because hard casting Holy Spirit as filler turned out to be a damage lost (because of the damage loss on the auto) or damage neutral in the long run. Paladin works with a priority system now (and the math behind it backs it up).

Reply:
Ah I just double checked now and they've added info w/r/t prioritising DM HS. Cool, easy.